projects/igniteui-angular/src/lib/list/list.component.spec.ts

```diff
@@ -611,13 +686,28 @@ describe('List', () => {
         expect(firstItem.panState).toBe(IgxListPanState.NONE);
         expect(leftPanTmpl.nativeElement.style.visibility).toBe('hidden');
         expect(rightPanTmpl.nativeElement.style.visibility).toBe('hidden');
+        expect(timesCalledStartPan).toBe(1);
```

Comment:
I can't see any action performed in this test. There seem to be mention of the cancel at all.